packages/webdriverio/src/commands/browser/newWindow.js

```diff
@@ -9,13 +9,14 @@
  * <example>
     :newWindowSync.js
     it('should open a new tab', () => {
-        browser.url('http://google.com')
+        browser.url('https://google.com')
         console.log(browser.getTitle()) // outputs: "Google"
 
         browser.newWindow('https://webdriver.io', 'WebdriverIO window', 'width=420,height=230,resizable,scrollbars=yes,status=1')
         console.log(browser.getTitle()) // outputs: "WebdriverIO · Next-gen browser automation test framework for Node.js"
 
         browser.closeWindow()
+        browser.switchWindow('https://google.com')
```

Comment:
This `switchWindow` call is not necessary as WebdriverIO automatically switches to the next window.

Reply:
@christian-bromann Indeed, `newWindow` switches to next window. But `closeWindow` does not switch back to old window.

Reply:
@drptbl sounds like we should fix that instead of updating the docs